src/Cmp/Queues/Domain/Event/JSONDomainEventFactory.php

```diff
@@ -22,7 +22,7 @@ public function create($json)
             throw new InvalidJSONDomainEventException("String is not valid JSON");
         }
 
-        if (!isset($domainEventArray['origin'], $domainEventArray['name'], $domainEventArray['version'], $domainEventArray['occurredOn'], $domainEventArray['body'])) {
```

Comment:
why this change?

Reply:
because if you check the class DomainEvent, body is optional. So this is actually a bug